### PR TITLE
backup.swarm

### PR DESCRIPTION
**Pull request by areed**

### CODE REVIEW

File: src/replicated.ts [hunk 1]
```diff
@@ -248,11 +248,21 @@ export interface ContentTrust {
 // backup
 /*
 type Backup struct {
-	Enabled         string `yaml:"enabled" json:"enabled"`
-	Hidden          string `yaml:"hidden" json:"hidden"`
-	PauseAll        bool   `yaml:"pause_all" json:"pause_all"` // deprecated
-	PauseContainers string `yaml:"pause_containers" json:"pause_containers"`
-	Script          string `yaml:"script" json:"script"`
+    Enabled         string       `yaml:"enabled" json:"enabled"`
```
Comment (divolgin) [inline src/replicated.ts]: is this just dead code?

**Review by divolgin — APPROVED** [review]

File: src/replicated.ts (hunk 1 above)
Comment (dexhorthy) [reply src/replicated.ts]: Yeah this whole file can probably be dumped. We we're gonna try to generate schema from typescript but I don't think it's worth it. We currently generate from golang

File: src/schemas/replicated.json [hunk 2]
```diff
@@ -1,1733 +1,657 @@
 {
-  "$schema": "http://json-schema.org/schema#",
```
Comment (dexhorthy) [inline src/schemas/replicated.json]: This file is unused, we transpiled it to typescript in `parsed.ts` to make it easier to include in the module.

This file can be deleted if you want to.

**Review by emosbaugh — COMMENTED** [review]
ill leave this one up to the pros

**Pull request merged**